client/components/__tests__/server.test.js

```diff
@@ -0,0 +1,13 @@
+import '@babel/polyfill';
```

Comment:
What is polyfill used for?

Reply:
It helps babel translate into ES5, in this case it helps with async + await.